docs/ABOUT.md

```diff
@@ -1,17 +1,20 @@
 # About
 
-Raku (formerly known as Perl 6) is a clean, modern, multi-paradigm language; it offers procedural, object-oriented AND functional programming methodologies.
-It is a supremely flexible language, adapting to your style of programming, whether that be quick one­liners for sysadmins, scripts to manage a database import, or the full stack of modules necessary to realise an entire website.
+I doubt you'll ever find a more expressive programming language. [Raku](https://raku.org/) is a modern, multi-paradigm language. Raku offers [procedural](https://docs.raku.org/language/functions), [object-oriented](https://docs.raku.org/language/objects) and [functional programming](https://docs.raku.org/language/functions#Functions_are_first-class_objects) methodologies.
```

Comment:
```suggestion
It may be difficult to find a programming language that is more expressive. [Raku](https://raku.org/) is a modern, multi-paradigm language. Raku offers [procedural](https://docs.raku.org/language/functions), [object-oriented](https://docs.raku.org/language/objects) and [functional programming](https://docs.raku.org/language/functions#Functions_are_first-class_objects) methodologies.
```